### PR TITLE
docs Rule based segments

### PR DESCRIPTION
**Pull request by jacob-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://63334456b16ae822e4febd4d--cozy-fox-0defec.netlify.app

File: docs/console-api/openapi/segments.js [hunk 1]
```diff
@@ -723,6 +745,155 @@ module.exports = {
...
+                      "type": "integer"
+                    },
+                    "message": {
+                      "type": "string"
+                    }
+                  },
+                  "x-examples": {
+                    "example-1": {
+                      "status": 400,
+                      "message": "Segment must be of type 'id_list' to be modified"
```
Comment (tore-statsig) [inline docs/console-api/openapi/segments.js:818]: should be rule_based

File: docs/console-api/openapi/segments.js [hunk 2]
```diff
@@ -723,6 +745,155 @@ module.exports = {
...
+                      "name": "App version check",
+                      "passPercentage": 10,
+                      "conditions": [
+                        {
+                          "type": "app_version",
+                          "operator": "version_gt",
+                          "targetValue": [
+                            "1.1.1"
+                          ]
+                        }
```
Comment (tore-statsig) [inline docs/console-api/openapi/segments.js:883]: where are we documenting all of the conditions that can be used across gates/segments/dyanmic configs/etc?

**Review by tore-statsig — APPROVED** [review]

File: docs/console-api/openapi/segments.js (hunk 2 above)
Comment (jacob-statsig) [reply docs/console-api/openapi/segments.js:883]: There is no current documentation. It is my highest priority as of now

File: docs/console-api/openapi/segments.js (hunk 1 above)
Comment (jacob-statsig) [reply docs/console-api/openapi/segments.js]: [f789c76](https://github.com/statsig-io/docs/pull/464/commits/f789c76dc3f10ac9b60d5cd3600e273944140e6a)

**Pull request merged**